build.sbt

```diff
@@ -178,6 +178,26 @@ lazy val tools = project
   )
   .dependsOn(core, clientJVM)
 
+lazy val tracing = project
+  .in(file("tracing"))
+  .enablePlugins(BuildInfoPlugin)
+  .settings(name := "caliban-tracing")
+  .settings(commonSettings)
+  .settings(
+    buildInfoPackage := "caliban.tracing",
+    buildInfoObject  := "BuildInfo"
+  )
+  .settings(
+    testFrameworks := Seq(new TestFramework("zio.test.sbt.ZTestFramework")),
+    libraryDependencies ++= Seq(
+      "dev.zio"         %% "zio-opentelemetry"         % "3.0.0-RC3",
```

Comment:
There was a RC4 published a few days ago

Reply:
Can you move the version to a variable?

Reply:
Done! 👍

Reply:
RC4 seems to be missing from Sonatype as well :(